test/Npgsql.Specification.Tests/NpgsqlDbFactoryFixture.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Data.Common;
+using AdoNet.Specification.Tests;
+
+namespace Npgsql.Specification.Tests
+{
+    public class NpgsqlDbFactoryFixture : IDbFactoryFixture
+    {
+        public DbProviderFactory Factory => NpgsqlFactory.Instance;
+
+        const string DefaultConnectionString =
+            "Server=localhost;User ID=npgsql_tests;Password=npgsql_tests;Database=npgsql_tests;Timeout=0;Command Timeout=0";
```

Comment:
(tiny) nit: We use `"Server=localhost;Username=npgsql_tests;Password=npgsql_tests";` in other test projects. Maybe align for consistency?

Reply:
That was actually a copy-paste from the main test project, but I updated both to use `Username` which looks nicer :)